docs/docs/contributing/eclipse.md

```diff
@@ -3,30 +3,55 @@ layout: doc-page
 title: Building Dotty with Eclipse
 ---
 
-Build setup
+Setup
 -----------
 
-1. Run `sbt ;managedSources;eclipse`
+You can setup Eclipse by using _[Scala IDE for Eclipse](http://scala-ide.org/)_ directly to run Dotty. Alternatively you can download 
+your Eclipse distribution of choice and enable the Scala IDE from the Eclipse Marketplace, separately.
 
-2. It is recommended to change the default output folder (in `Properties > java
+Assuming you have cloned dotty's repository from Github (and run `sbt managedSources` as described in 
+[Getting Started](getting-started.md)) you can now proceed to set up Dotty with Eclipse. In addition to the previous
+command you need to run `sbt eclipse` to generate the project structure for Eclipse.
```

Comment:
Ah I see now, thanks. But I would prefer to not leave this critical command inside a line of text, I think it'd be better to have it in a real code block.